### PR TITLE
apply new pom structure #391

### PR DESCRIPTION
**Pull request by bttanakagn**
Please review #391.

### CODE REVIEW

File: terasoluna-tourreservation-web/pom.xml [hunk 1]
```diff
@@ -63,11 +63,18 @@
...
         <dependency>
             <groupId>org.terasoluna.gfw</groupId>
-            <artifactId>terasoluna-gfw-security-web</artifactId>
+            <artifactId>terasoluna-gfw-security-web-dependencies</artifactId>
+            <type>pom</type>
+        </dependency>
+        <dependency>
+            <groupId>org.terasoluna.gfw</groupId>
+            <artifactId>terasoluna-gfw-security-core-dependencies</artifactId>
+            <type>pom</type>
```
Comment (btyoshikawaa) [inline terasoluna-tourreservation-web/pom.xml]: - [ ] Is this dependency necessary in web project? I seems that domain project already depend on this.

Comment (bttanakagn) [inline terasoluna-tourreservation-web/pom.xml]: I fixed this dependency at 34216c80519a5180417357b253af19afcfa605db. please confirm it.

File: pom.xml [hunk 2]
```diff
@@ -176,18 +215,33 @@
...
+        <formatter-maven-plugin.version>0.5.2</formatter-maven-plugin.version>
+        <com.google.code.maven-license-plugin.version>1.4.0</com.google.code.maven-license-plugin.version>
+        <!-- == Other Properties== -->
         <encoding>UTF-8</encoding>
         <java-version>1.8</java-version>
-        <org.codehaus.mojo.build-helper-maven-plugi.version>1.9.1</org.codehaus.mojo.build-helper-maven-plugi.version>
         <postgresql.version>9.4-1206-jdbc41</postgresql.version>
         <jasperreports.version>3.5.1</jasperreports.version>
+        <selenium.version>2.46.0</selenium.version>
+        <project.root.basedir>${project.basedir}</project.root.basedir>
```
Comment (btyoshikawaa) [inline pom.xml]: @bttanakagn 
I think that it is more better to be categorized as follows. What do you think?
* Project Properties ( So-called `Other Properties` )
* Dependency Versions and Properties
* Maven Plugin Versions and Properties

Comment (bttanakagn) [inline pom.xml]: @btyoshikawaa 
That's bright idea. I will modify properties that way.

Comment (bttanakagn) [inline pom.xml]: @btyoshikawaa 
Should I create issue about property category and fix at there?
Those are many pom.xml of other projects should be applied the same way.

Comment (btyoshikawaa) [inline pom.xml]: @bttanakagn 
Let's that way. Creating issue or Reopening issue, either is fine. Thanks.

Comment (bttanakagn) [inline pom.xml]: @btyoshikawaa 
I fixed separate comment. Please confirm it.
In other projects, I'll reopen issue and fix that comment.

**Pull request merged**